templates.dist/estate/default_detail.php

```diff
@@ -32,7 +35,13 @@
 <div class="oo-detailview">
 	<?php
 	$pEstates->resetEstateIterator();
-	while ( $currentEstate = $pEstates->estateIterator() ) { ?>
+	while ( $currentEstate = $pEstates->estateIterator() ) {
+		if (get_option('onoffice-settings-title-and-description') == 1)
+		{
+			$estateListHandle = new EstateListHandle();
+			$currentEstate = $estateListHandle->handleRecord($currentEstate);
```

Comment:
Could you explain to me what this does? I don't understand what problem it solves.

Reply:
i check when user selected option "Do not modify", i will change value of field "objekttitel" =  value of custom field " `onoffice_titel` / `onoffice_title`"